### DISCUSSION TITLE
problem in show my jupyter notebook

### DISCUSSION THREAD

**Discussion by Yasser-Cloud** [community/community#124709] [Codespaces]
### Select Topic Area

Question

### Body

can't open my Jupyuter Notebook on GitHub after using Codespace. I find that 
![image](https://github.com/community/community/assets/52430441/f10140d5-74f3-48b3-bfb8-d5781e8842ce)

**Comment by Such-13** [comment]
@Github please rectify or provide a solution 

Same problem

**Reply by Yasser-Cloud** [reply]
Issue with GitHub but solved now
